src/routes/common/logging/logging.service.spec.ts

```diff
@@ -0,0 +1,69 @@
+import { ClsService } from 'nestjs-cls';
+import { RequestScopedLoggingService } from './logging.service';
+import * as winston from 'winston';
+
+const mockClsService = {
+  getId: jest.fn(() => '123-456'),
+} as unknown as ClsService;
+
+jest.mock('winston');
+
+describe('RequestScopedLoggingService', () => {
+  const infoMock = jest.fn();
+  const debugMock = jest.fn();
+  const errorMock = jest.fn();
+  const warnMock = jest.fn();
+  let loggingService: RequestScopedLoggingService;
+
+  beforeAll(() => {
+    jest.useFakeTimers();
+    jest.setSystemTime(new Date('2023-01-01'));
+    jest.spyOn(winston, 'info').mockImplementation(infoMock);
+    jest.spyOn(winston, 'debug').mockImplementation(debugMock);
+    jest.spyOn(winston, 'error').mockImplementation(errorMock);
+    jest.spyOn(winston, 'warn').mockImplementation(warnMock);
+  });
+  beforeEach(() => {
+    loggingService = new RequestScopedLoggingService(mockClsService as any);
```

Comment:
You don't need to cast to `any` as you have casted `mockClsService` on line 7.

```suggestion
    loggingService = new RequestScopedLoggingService(mockClsService);
```

Reply:
Nit: you also need not reassign this before each test as you are calling `clearAllMocks` between each one.